flycheck.el

```diff
@@ -3942,8 +3967,10 @@ Return a list with the contents of the table cell."
                    column 'flycheck-error-list-column-number)
                   (flycheck-error-list-make-cell
                    (symbol-name (flycheck-error-level error)) level-face)
-                  (flycheck-error-list-make-cell
-                   id-str 'flycheck-error-list-id id-str)
+                  ;; Error ID has a button to explain the error
+                  (list id-str
+                        'type 'flycheck-error-list-explain-error
```

Comment:
Aaaah, I get it now.  I didn't realize that the button wasn't getting a link face (the one with a blue underline), sorry.  My proposal was to remove that face and make the button inactive when there was no explainer… but now I realize that there is currently no face :)

I understand now; sorry for the back and forth.  I think the current implementation is fine, and we can in th future possibly add a hyperlink to checkers that have an explainer.